src/dodal/common/visit.py

```diff
@@ -78,7 +78,7 @@ async def _identifier_from_response(
         ):
             response.raise_for_status()
             json = await response.json()
-            return DataCollectionIdentifier.model_validate_json(json)
+            return DataCollectionIdentifier.model_validate(json)
```

Comment:
`model_validate_json` is for turning a JSON string into the object. `model_validate` is for turning a Python object. `response.json()` seems to convert the JSON string returned by the numTracker service into a dict.